### ISSUE TITLE
LibJS: "Dynamic Imports are not allowed" on https://polar.sh/

### ISSUE THREAD

**Issue by awesomekling** [SerenityOS/serenity#21568]
When trying to load https://polar.sh/ I get the following error:

```
Unhandled JavaScript exception (in promise): [Error] Dynamic Imports are not allowed
  <unknown> at preview_script0.NW6VTZ6X.mjs:1:487
  <unknown> at chunk-LC66P5ZC.mjs:9:36663
  <unknown> at preview_script0.NW6VTZ6X.mjs:1:2392
  <unknown> at preview_script0.NW6VTZ6X.mjs:1:2577
  <unknown> at preview_script0.NW6VTZ6X.mjs:1:2577
  <unknown> at :0:0
```

## Funding

- We're currently trying [Polar.sh](https://polar.sh/SerenityOS) where you can help fund the work on this issue.
- Contributor(s) receive the funding once the issue is completed & confirmed.

<a href="https://polar.sh/SerenityOS/serenity/issues/21568">
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/api/github/SerenityOS/serenity/issues/21568/pledge.svg?darkmode=1">
  <img alt="Fund with Polar" src="https://polar.sh/api/github/SerenityOS/serenity/issues/21568/pledge.svg">
</picture>
</a>

**Comment by stelar7** [comment]
@awesomekling this seems to be fixed as of https://github.com/SerenityOS/serenity/pull/21967 ?

**Comment by awesomekling** [comment]
> @awesomekling this seems to be fixed as of #21967 ?

True, it is indeed fixed! :)
